node/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ All notable changes to this project will be documented in this file.  The format
   `network.estimator_weights`.
 * Added `consensus.highway.max_request_batch_size` configuration parameter. Defaults to 20.
 * New histogram metrics `deploy_acceptor_accepted_deploy` and `deploy_acceptor_rejected_deploy` that track how long the initial verification took.
+* New JSON-RPC endpoint `get-validator-info` and REST endpoint `validator-changes`  that return the changes in the validator set between two eras.
```

Comment:
The `get-validator-info` method is not listed in the response to the `rpc.discover` query. Please consider updating the `OPEN_RPC_SCHEMA` static.

Reply:
```suggestion
* New JSON-RPC endpoint `get-validator-info` and REST endpoint `validator-changes` that return the changes in the validator set between two eras.
```

Reply:
Also, I'm having trouble using the new RPC endpoint:
```
$ curl -d '{"id":"1", "jsonrpc":"2.0", "method":"get-validator-info"}' -H "Content-Type: application/json" -X POST http://172.16.0.8:11103/rpc/
{"jsonrpc":"2.0","id":"1","error":{"code":-32601,"message":"Method not found","data":null}}
```

For REST, it's working correctly:
```
$ curl http://172.16.0.8:14103/validator-changes
{"api_version":"1.0.0","changes_to_validators":[]}
```

Reply:
Ok, ok, so I see it's available as a separate path. Please scrap the comments above if this is intended ;-)

```
$ curl http://172.16.0.8:14103/validator-changes
{"api_version":"1.0.0","changes_to_validators":[]}
```